integration-cli/requirements.go

```diff
@@ -32,6 +33,13 @@ var (
 		func() bool { return supportsExec },
 		"Test requires 'docker exec' capabilities on the tested daemon.",
 	}
+	Network = TestRequirement{
+		func() bool {
+			_, err := http.Get("http://google.com")
```

Comment:
I'm pretty sure you are going to leak here because you are not closing the connection